Makefile

```
@@ -37,7 +37,7 @@ help: ## Display this help.
 ##@ Development
 
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./pkg/apis/..." output:crd:artifacts:config=manifests/base/crds
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=training-operator webhook paths="./pkg/..." output:crd:artifacts:config=manifests/base/crds output:rbac:artifacts:config=manifests/base/rbac
```

```suggestion
	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./pkg/..." output:crd:artifacts:config=manifests/base/crds output:rbac:artifacts:config=manifests/base/rbac
```

Can we keep using `manager-role` for backward compatibility?

---

Actually, RBAC manifests were not generated by controller-gen previously（Because the path=./pkg/apis/... does not include the kubebuilder RBAC flag, which is in ./pkg/controller.v1/...）, and the role name was training-operator. Therefore, to maintain backward compatibility, replace manager-role to training-operator.
https://github.com/kubeflow/training-operator/blob/fc8a64464f364565fcd399d9d7724201e48b0dfa/manifests/base/cluster-role.yaml#L2-L7

---

Oh, I see. Thank you for the clarification!